docs/configuration.md

```diff
@@ -113,12 +96,12 @@ The mongodb post used by udata.
 
 **default**: `udata`
 
-The mongodb database used by udata. When testing, the test database will use the same name suffixed by `-test`
+The mongodb database used by udata.
+During tests, the test database will use the same name suffixed by `-test`
 
-See [the official Flask-MongoEngine documentation](https://flask-mongoengine.readthedocs.org/en/latest/)
+See [the official Flask-MongoEngine documentation][flask-mongoengine-doc]
```

Comment:
No that kind of links is [valid commonmark(down)](http://commonmark.org/help/), they are defined at the bottom of the document.